src/card/basic/card.directive.js

```diff
@@ -3,26 +3,140 @@
  * @name patternfly.card.directive:pfCard
  * @restrict A
  * @element ANY
- * @param {headTitle=} Title for the card - required
- * @param {subTitle=} Subtitle for the card - optional
- * @param {showTopBorder=} Show Top Border, true shows top border, false (default) hides top border - optional
- *
+ * @param {string} headTitle Title for the card
+ * @param {string=} subTitle Sub-Title for the card
+ * @param {boolean=} showTopBorder Show/Hide the blue top border. True shows top border, false (default) hides top border
+ * @param {boolean=} showBottomBorder Show/Hide the bottom grey line between the title and sub-title.
```

Comment:
Might want to call this showTitlesSeparator or something. I expected this to mean show a border at the bottom of the card.